packages/frontend/src/components/common/SpaceActionModal/CreateSpaceAddUser.tsx

```diff
@@ -0,0 +1,188 @@
+import { Classes, Label } from '@blueprintjs/core';
```

Comment:
Files copied and updated from edit ShareSpaceModal... I tried to refactor and make it generic, but it was too messy (one relies in existing space with mutations, the other isn't...)